src/index.js

```diff
@@ -1,14 +1,86 @@
 import './styles/main.scss';
 import showPop from './popup.js';
+import movieImg from './assets/video-player.png';
+
+// disable linters for too many classes
+
+class Likes {
```

Comment:
### Optional
- [ ] It would be better if we create a separate file for each of the classes, in this way we try and implement Single Responsibility Principle

Reply:
New file created for likeComments class